pkgs/development/python-modules/wandb/default.nix

```
@@ -0,0 +1,71 @@
+{ buildPythonPackage
+, click
+, configparser
+, docker_pycreds
+, fetchFromGitHub
+, git
+, GitPython
+, lib
+, pathtools
+, promise
+, protobuf
+, psutil
+, python
+, python-dateutil
+, pyyaml
+, requests
+, sentry-sdk
+, shortuuid
+, subprocess32 ? null
+, yaspin
+}:
+
+buildPythonPackage rec {
+  pname = "wandb";
+  version = "0.12.3";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = "client";
+    rev = "v${version}";
+    sha256 = "08m6mcskmv4cp4zm53cqgbnbwlk7jlqsrq1gxbwzsqwngw2hcwjq";
+  };
+
+  # The wandb requirements.txt does not distinguish python2/3 dependencies. We
+  # need to drop the subprocess32 dependency when building for python3.
+  patchPhase = lib.optional python.isPy3 ''
+    sed -i 's/^subprocess32>=.*//' requirements.txt
+  '';
+
+  # git is not a setup.py dependency of wandb, but wandb does expect git to be
+  # in PATH.
+  propagatedBuildInputs = [
+    click
+    configparser
+    docker_pycreds
+    git
+    GitPython
+    pathtools
+    promise
+    protobuf
+    psutil
+    python-dateutil
+    pyyaml
+    requests
+    sentry-sdk
+    shortuuid
+    yaspin
+  ] ++ lib.optionals python.isPy2 [ subprocess32 ];
```

py2 is no longer supported.

```suggestion
  ];
```

---

Do you mean no longer supported by NixOS or no longer supported by wandb?